app/jobs/warm_bgs_caches_job.rb

```diff
@@ -116,7 +116,7 @@ def warm_poa_for_oldest_cached_records
     oldest_bgs_poa_records.limit(LIMITS[:OLDEST_CACHED]).each do |bgs_poa|
       begin
         bgs_poa.save_with_updated_bgs_record! if bgs_poa.stale_attributes?
-      rescue Errno::ECONNRESET, Savon::HTTPError
+      rescue StandardError
         # no nothing
```

Comment:
```suggestion
        # do nothing
```

We were doing nothing for those 2 specific errors that we know about. If you're generalizing the rescue, should we log it in case there are other error types:
```ruby
      rescue Errno::ECONNRESET, Savon::HTTPError
        # do nothing
      rescue StandardError => error
        capture_exception(error: error)
```

Reply:
I made this change